File: src/backend/core/DTO/AuthDTO.cs
namespace SnippetVault {
using System;
    

public class SignInDTO {
    public string userName {get; set;}
    public string password {get; set;}
}

public class UpdatePwDTO {
    public SignInDTO signIn {get; set;}
    public string newPw {get; set;}
}

public class SignInAdminDTO {
    public string userName {get; set; }
    public string password1 {get; set; }
    public string password2 {get; set; }
}

public class UpdatePwAdminDTO {
    public SignInAdminDTO signIn {get; set;}
    public string newPw {get; set;}
}

public class SignInSuccessDTO {
    public int userId {get; set; }
    public string accessToken {get; set; }
}

}

File: src/backend/main/tech/sozonov/SnippetVault/core/DTO/AuthDTO.java
package tech.sozonov.SnippetVault.core.DTO;


public class AuthDTO {

public static class SignInSuccess {
    public int userId;
}

public static class SignIn {
    public String userName;
    public String password;
}

public static class ChangePw {
    public SignIn signIn;
    public String newPw;
}

public static class SignInAdmin {
    public String userName;
    public String password1;
    public String password2;
}

public static class ChangePwAdmin {
    public SignInAdmin signIn;
    public String newPw;
}

}
